Formula/vert.rb

```diff
@@ -0,0 +1,27 @@
+class Vert < Formula
+  desc "Command-line version testing"
+  homepage "https://github.com/Masterminds/vert"
+  url "https://github.com/Masterminds/vert/archive/v0.1.0.tar.gz"
+  sha256 "96e22de4c03c0a5ae1afb26c717f211c85dd74c8b7a9605ff525c87e66d19007"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/Masterminds/vert").install buildpath.children
+    cd "src/github.com/Masterminds/vert" do
+      system "dep", "ensure"
+      system "make", "build"
+      bin.install "vert"
```

Comment:
Would be surprised if a `prefix.install_metafiles` line was not required inside this block.

Reply:
@DomT4 What does `prefix.install_metafiles` do? I'm not finding it in the docs.

Reply:
It looks like it copies the README and LICENSE files?

Reply:
Yep!  It's run automatically in the root directory, so we only need to call it manually when everything is in a subdirectory.